modules/data.atmosphere/R/solar_angle.R

```diff
@@ -2,6 +2,50 @@
 #'
 #' For explanations of formulae, see https://web.archive.org/web/20180307133425/http://www.itacanet.org/the-sun-as-a-source-of-energy/part-3-calculating-solar-angles/
```

Comment:
CRAN is picky about URLs—can you run `urlchecker::url_check("modules/data.atmosphere")` to check that this is OK?